pvlib/spectrum/mismatch.py

```diff
@@ -571,3 +571,126 @@ def spectral_factor_caballero(precipitable_water, airmass_absolute, aod500,
     )
     modifier = f_AM + f_AOD + f_PW  # Eq 5
     return modifier
+
+
+def spectral_factor_pelland(airmass_absolute, clearsky_index,
+                            module_type=None, coefficients=None):
+    r"""
+    Estimate a technology-specific spectral mismatch modifier from
+    airmass and clear sky index using the Pelland model, which takes the
+    following form:
+
+    .. math::
```

Comment:
One of the mentors commented on the last meeting that equations are usually on the notes section. Can this be moved there?

Reply:
@echedey-ls Yes I was going to ask about this as I thought they did but then couldn't remember when updating the script. I will make the change.